### ISSUE TITLE
Is there any way to have a role to match any single word?

### ISSUE THREAD

**Issue by dylan-chong** [dictation-toolbox/aenea#154]
If i want to say `printf`, then if i have a grammar that uses `Dictation()` somewhere in it, then i will have to pause after saying `print`  , then say `foxtrot`.

Is there any way i can get the Dictation to match only a single word, so i can say this without any pauses: `word print foxtrot` to type out `printf`?

**Comment by dylan-chong** [comment]
If not, i might hack something up using http://www.mieliestronk.com/corncob_lowercase.txt

**Comment by RyanQuey** [comment]
Yeah, this would be a nice feature. Let me know if you find something

**Comment by calmofthestorm** [comment]
I don't know a way to do this with Natlink/Dragonfly. IME, Dictation will match all words from when it starts until a pause, and there's no way to make it less eager. If you can get away with a small set of things rather than general purpose dictation, there are options, but that file you link is huge; I wonder how well Dragon could handle it.

**Comment by dylan-chong** [comment]
Ill assume it won't be worth the effort trying to use that massive file because i found a better solution 
to my specific problem

My problem is that i have to pause lots after doing a wildcard `Dictation`. My (incorrect) assumption was that you had to stop talking after a Dictation. It turns out that dragon lets the wildcard end if it expects and you say some command afterwards. e.g. I can say `backtick Proper dictation backtick` to type `\`Dictation\`` (where Proper is the wildcard).

The only annoying thing is i don't want to end the Dictation when i'm writing some text. I might say: `Sentence The first word in the nato alphabet is alpha` and dragon will type it out as `The first word in the nato alphabet isa`. I'm not sure on a good way to solve this problem other than using different trigger words for the wildcard...

**Comment by Versatilus** [comment]
The way Caster tackles this problem is by specifying an optional word/sound
at the end of the spec for dictation.

https://github.com/synkarius/caster/blob/master/caster/lib/ccr/core/nav.py#L149

**Comment by dylan-chong** [comment]
Ah yes i was considering that idea. I'll certainly try it out, although it may have an impact on speed. Then again it might be worth it. I'll give it a go and let you know how it works out
